scripts/get-hypper

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env bash
+# Copyright The Helm Authors, SUSE LLC
+# Copyright (C) 2014-2016, Matt Butcher and Matt Farina
+# Copyright (C) 2016, Hewlett Packard Enterprise Development LP
+# Copyright (C) 2015, Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The install script is based off of the MIT-licensed script from glide,
```

Comment:
While I think it's great that you pulled the script from Glide... you might try pulling it from one the of the rancher projects. I think the people who worked on that stuff were better at shell scripts than Butcher or I were when we did the Glide work.

Have you looked at https://github.com/rancher/rio/blob/master/install.sh

Reply:
I did not pull it from glide, I lifted it from helm directly to be as close as possible (and because I have used it before so I had read it already :P)

Its a very simple script, and most of the install ones looks very much alike (like the one you linked) so I dont think there is much room here to improve

Reply:
Ha. I didn't realize that Helm had pulled it from glide. I didn't realize the provenance.

Reply:
fixed!